make.js

```diff
@@ -145,8 +130,8 @@ target.gendocs = function() {
 // ex: node make.js build
 // ex: node make.js build --task ShellScript
 //
-target.build = function() {
-    target.clean();
+CLI.build = function(/** @type {{ suite: string; node: string; … }} */ argv) {
```

Comment:
```suggestion
CLI.build = function() {
```
Used here only as param for `CLI.clean` function, where not used (see previous comment)